source/core/vireo.loader.staticHelpers.js

```diff
@@ -7,8 +11,8 @@ const encodeIdentifier = function (str) {
         codePoint = str.charCodeAt(0),
         ch = str.charAt(0);
 
-    // First character must be encoded if is not a letter [A-Za-z]
-    if (!(codePoint >= 0x41 && codePoint <= 0x5A) && !(codePoint >= 0x61 && codePoint <= 0x7A)) {
+    // First character must be encoded if is not a letter [A-Za-z] or a non-ascii character
+    if (!isCharacterAlphabetOrNonAscii(codePoint)) {
```

Comment:
ESLint thinks my code's cyclomatic complexity is high. Broke it out into a function. 🙄

Reply:
Looks like a good clean code change to me. With a self-documenting function name, the comment isn't really useful anymore if you want to get rid of it.